cmd/handlers.c

```diff
@@ -47,6 +53,14 @@ const CmdList commands[] = {
   {CMD_REST_REQUEST,    "REST_REQ",       REST_Request},
   {CMD_REST_SETHEADER,  "REST_SETHDR",    REST_SetHeader},
 #endif
+#ifdef TCP
+  {CMD_TCP_SETUP,      "TCP_SETUP",     TCP_Setup},
+  {CMD_TCP_SEND,       "TCP_SEND",      TCP_Send},
```

Comment:
Left as it is. The ESP8266 disconnects a TCP socket in client mode after data was sent, or (if data is expected) after data was received. TCP server keeps the connection open until the client closes it (not the best solution, I know) and for UDP there is no connect/disconnect anyway.